test/Serialization/Fixtures/SerializableObjectUsingTrait.php

```diff
@@ -2,7 +2,7 @@
 
 namespace BroadwaySerialization\Test\Serialization\Fixtures;
 
-use Broadway\Serializer\SerializableInterface;
+use Broadway\Serializer\Serializable as SerializableInterface;
```

Comment:
I was going for the smoothest update path for the user. If we rename the trait it has to be updated in all of the read models/value objects that use it. 

These are also test fixtures so I'm not sure if it's worth the effort of the end user for something internal?